### PR TITLE
[New Rule] Access to SystemKey via Hexdump

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/814

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/credential_access_systemkey_dumping.toml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+references = ["https://github.com/AlessandroZ/LaZagne/blob/master/Mac/lazagne/softwares/system/chainbreaker.py"]
+risk_score = 73
+rule_id = "d75991f2-b989-419d-b797-ac1e54ec2d61"
+severity = "high"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Credential Access"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+  process.name:hexdump and  process.args:"/private/var/db/SystemKey"
```
Comment (bm11100) [inline rules/macos/credential_access_systemkey_dumping.toml:27]:
```suggestion
  process.name:hexdump and process.args:"/private/var/db/SystemKey"
```

File: rules/macos/credential_access_systemkey_dumping.toml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+creation_date = "2020/01/07"
+maturity = "production"
+updated_date = "2020/01/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may collect the keychain storage data from a system to acquire credentials. Keychains are the built-in way
+for macOS to keep track of users' passwords and credentials for many services and features such as WiFi passwords,
+websites, secure notes, certificates, and Kerberos.
```
Comment (bm11100) [inline rules/macos/credential_access_systemkey_dumping.toml:11]:
```suggestion
Keychains are the built-in way for macOS to keep track of users' passwords and credentials for many services and features, including Wi-Fi and website passwords, secure notes, certificates, and Kerberos. Adversaries may collect the keychain storage data from a system to acquire credentials. 
```

File: rules/macos/credential_access_systemkey_dumping.toml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Access to SystemKey via Hexdump"
```
Comment (bm11100) [inline rules/macos/credential_access_systemkey_dumping.toml:17]:
```suggestion
name = "SystemKey Access via Hexdump"
```

File: rules/macos/credential_access_systemkey_dumping.toml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/01/07"
+maturity = "production"
+updated_date = "2020/01/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Keychains are the built-in way for macOS to keep track of users' passwords and credentials for many services and features, including Wi-Fi and website passwords, secure notes, certificates, and Kerberos. Adversaries may collect the keychain storage data from a system to acquire credentials.
```
Comment (bm11100) [inline rules/macos/credential_access_systemkey_dumping.toml:9]: just needs a re-lint, otherwise LGTM

**Review by bm11100 — APPROVED** [review]

**Review by brokensound77 — APPROVED** [review]
:shipit:

**Pull request merged**